app/.env.example

```
@@ -16,6 +16,16 @@ TOGETHER_API_KEY=TOGETHER_API_KEY
 
 SECRET_KEY=SECRETKEYSECRETKEYSECRETKEYSECRE
 
+
+MONGO_SCHEME=mongodb
+MONGO_HOST=localhost
+MONGO_DB="devdox"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)